src/scripts/make-configs.sh

```diff
@@ -64,11 +64,11 @@ server.ssl.key-alias=internal-sys-dev
 server.ssl.key-store-password=$KEYSTORE_PASSWORD
 server.ssl.trust-store=file:target/certs/system/DVP-NONPROD-truststore.jks
 server.ssl.trust-store-password=$KEYSTORE_PASSWORD
-server.ssl.client-auth=want
+server.ssl.ppmsClient-auth=want
```

Comment:
This should still be `client`

Reply:
Should the Client SSL below (line 71) also be `client`?  (Instead of ppmsClient)

Reply:
yes